src/IceRpc.Templates/Templates/IceRpc-Protobuf-Client/IceRpc-Protobuf-Client.csproj

```diff
@@ -28,6 +28,10 @@
     <PackageReference Include="Microsoft.Extensions.Logging" Version="9.0.*-preview.*" />
     <PackageReference Include="Microsoft.Extensions.Logging.Console" Version="9.0.*-preview.*" />
     <!--#endif-->
+    <!--#if (framework=="net8.0" && transport=="quic")-->
```

Comment:
I am wondering if we should update the templates to configure "secure" all the time, even with Slic.

I find the current model  surprising, with QUIC secure and Slic non-secure.